introspection-engine/introspection-engine-tests/tests/relations/postgres.rs

```diff
@@ -119,7 +119,7 @@ async fn relations_should_avoid_name_clashes_2(api: &TestApi) -> TestResult {
           id                   Int @id @default(autoincrement())
           y                    Int
           y_x_yToy             y   @relation("x_yToy", fields: [y], references: [id], onDelete: NoAction, onUpdate: NoAction)
-          y_xToy_fk_x_1_fk_x_2 y[] @relation("xToy_fk_x_1_fk_x_2")
+          y_y_fk_x_1_fk_x_2Tox y[] @relation("y_fk_x_1_fk_x_2Tox")
```

Comment:
These names are the same, they just now always start from the referencing side of the relation, instead of the lexicographically smaller side.